scripts/bootstrap.sh

```diff
@@ -1,18 +1,26 @@
 #!/bin/bash
 set -euo pipefail
 
+BACKEND_SERVER_DIR="${BACKEND_SERVER_DIR:-/srv/backend-server}"
+
+if test "$PWD" != "$BACKEND_SERVER_DIR"; then
+  echo "You must run this script from a checkout of backend-server located at $BACKEND_SERVER_DIR"
+  exit 1
+fi
+
 if test -z "$1"
 then
   echo "Please specify your backend when running this file. For example:"
-  echo "./bootstrap.sh my_backend";
+  echo "$0 my_backend";
   exit 1
 fi
 
 echo "BACKEND_JUST=$1" > .env
 
 if [[ "$OSTYPE" == "linux-gnu"* ]]
 then
-  mv bin/just-1.14.0-x86_64-unknown-linux-musl /usr/local/bin/just
+  echo "Installing just in /usr/local/bin/just"
+  cp "$PWD/bin/just-1.14.0-x86_64-unknown-linux-musl" /usr/local/bin/just
   chmod 0755 /usr/local/bin/just
```

Comment:
open question - should we be doing this with `install` instead of `cp && chmod`? https://www.man7.org/linux/man-pages/man1/install.1.html

Reply:
Good idea